models/demos/wormhole/mamba/demo/demo.py

```diff
@@ -369,7 +369,7 @@ def callback(token: torch.Tensor, inference_time: float) -> None:
     )
     logger.info(f"Time to first token: {(1e3 * measurements['prefill_decode_t/s/u']):.2f} ms")
 
-    chunk_size_to_prefill_targets_tok_per_s = {32: 135.0, 128: 270.0}  # perf is different for different chunk sizes
+    chunk_size_to_prefill_targets_tok_per_s = {32: 120.0, 128: 270.0}  # perf is different for different chunk sizes
```

Comment:
perf regression?

Reply:
@uaydonat Was hitting this assert on a lab machine, but looks like nightly pipeline is still hitting the expected perf: [pipeline](https://github.com/tenstorrent/tt-metal/actions/runs/10673849275/job/29583646502#step:9:652)

I think it makes sense to revert this change.